src/components/CardListItem.js

```diff
@@ -118,17 +133,20 @@ export const CardListItem = memo(({ navigation, horizontal, item, sue }) => {
       onPress={() => navigation && navigation.push(name, params)}
       disabled={!navigation}
     >
-      <Card containerStyle={[styles.container, !!containerStyle && containerStyle]}>
-        <View
-          style={[
-            stylesWithProps({ horizontal }).contentContainer,
-            !!contentContainerStyle && contentContainerStyle,
-            sue && styles.sueContentContainer
-          ]}
-        >
-          {renderCardContent(item, horizontal, sue)}
-        </View>
-      </Card>
+      <View>
+        <Card containerStyle={[styles.container, containerStyle]}>
+          <View
+            style={[
+              stylesWithProps({ horizontal }).contentContainer,
+              contentContainerStyle,
+              sue && styles.sueContentContainer
+            ]}
+          >
+            {renderCardContent(item, horizontal, noOvertitle, bigTitle, sue)}
+          </View>
```

Comment:
The Title of the ListItem is not shown with a correct padding. Also the subtitle is not visible.

![Simulator Screenshot - iPhone 15 Pro Max - 2024-07-29 at 09 36 19](https://github.com/user-attachments/assets/d4121ad5-e918-4c65-b8b9-7b76c25d1ea6)

Reply:
ok

<img width="380" alt="image" src="https://github.com/user-attachments/assets/a16d2fc0-80f9-4047-910a-ed5dcab257d8">